src/Simulation/Common/QubitManagerTrackingScope.cs

```diff
@@ -176,7 +176,7 @@ public virtual Qubit Borrow()
             return base.Borrow(1, ConstructExcludedQubitsArray())[0];
         }
 
-        public virtual IQArray<Qubit> Borrow(long numToBorrow)
```

Comment:
Keeping this virtual is important for exploration experiments and layout benchmarks.